asset/quickstart/list-assets/main.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START asset_quickstart_list_assets]
+
+// Sample asset-quickstart list assets.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"google.golang.org/api/iterator"
+
+	asset "cloud.google.com/go/asset/apiv1p5beta1"
+	assetpb "google.golang.org/genproto/googleapis/cloud/asset/v1p5beta1"
+)
+
+func main() {
+	ctx := context.Background()
+	client, err := asset.NewClient(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
+	assetType := "storage.googleapis.com/Bucket"
+	req := &assetpb.ListAssetsRequest{
+		Parent:      fmt.Sprintf("projects/%s", projectID),
+		AssetTypes:  []string{assetType},
+		ContentType: assetpb.ContentType_RESOURCE,
+	}
+
+	// Call ListAssets API to get an asset iterator.
+	assetIterator := client.ListAssets(ctx, req)
+
+	// Traverse and print the listed assets in response.
+	for {
+		response, err := assetIterator.Next()
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			log.Fatal(err)
+		}
+		fmt.Print(response)
```

Comment:
```suggestion
		fmt.Println(response)
```

That way different assets are printed on separate lines.

Reply:
Done. Thanks!